src/app/home/events/events.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="event-container section center-align">
+  <div class="row event-container">
+    <h3 class="section-title">{{ 'HOME.EVENT.EVENTS' | translate }}</h3>
+    <div class="container-cards center-align">
+      <div *ngFor="let event of events" class="col s12 m4">
+        <div class="card">
+          <div class="column">
+            <div class="card-image image-item image-container">
+              <h1 class="title-event">{{ event.name }}</h1>
```

Comment:
You can use h5 instead of changing the font size of this h1. Size of h5 is 1.64rem which is 24.6px in desktop.